### PR TITLE
Edit Conflicts

### PR DESCRIPTION
**Pull request by temshik**
Resolve conflicts and use clean-up

### CODE REVIEW

File: src/Application/Interfaces/IRepositories/IGenericRepository.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿using System.Linq.Expressions;
 
-namespace Application.Interfaces
+namespace Application.Interfaces.IRepositories
 {
     public interface IGenericRepository<T> where T : class
```
Comment (svadrif) [inline src/Application/Interfaces/IRepositories/IGenericRepository.cs:5]: you also should add default value for tracking param in GetAll and Search methods

File: src/Infrastructure/Services/AppUserService.cs [hunk 2]
```diff
@@ -23,10 +25,11 @@ public async Task<Guid> AddAsync(AddAppUserDTO appUserDTO)
             return appUser.Id;
         }
 
-        public async Task<IEnumerable<GetAppUserDTO>> GetAllAsync()
+        public async Task<PagedList<GetAppUserDTO>> GetPagedAsync(PagedQueryBase query)
         {
-            var appUsers = await _unitOfWork.AppUsers.GetAllAsync();
-            return _mapper.Map<IEnumerable<GetAppUserDTO>>(appUsers);
+            var appUsers = await _unitOfWork.AppUsers.GetPagedAsync(query);
+            var appUsersDTO = new PagedList<GetAppUserDTO>(_mapper.Map<List<GetAppUserDTO>>(appUsers), appUsers.TotalCount, appUsers.CurrentPage, appUsers.PageSize);
```
Comment (svadrif) [inline src/Infrastructure/Services/AppUserService.cs:31]: in order to make code more readable create another var and assing this "_mapper.Map<List<GetAppUserDTO>>(appUsers)" to it.

This recommendation applies to other identical sections of code as well.

File: src/Infrastructure/Services/BookingService.cs [hunk 3]
```diff
@@ -49,8 +52,8 @@ public async Task<bool> RemoveAsync(Guid Id)
 
         public async Task<IEnumerable<GetBookingDTO>> SearchByUserIdAsync(Guid UserId)
         {
-            var bookings = await _unitOfWork.Bookings.Search(c => c.UserId.Contains(UserId));
-            if (bookings = null)
+            var bookings = _unitOfWork.Bookings.Search(c => c.UserId.Equals(UserId), false);
```
Comment (svadrif) [inline src/Infrastructure/Services/BookingService.cs:55]: As I recommended before, add default value for tracking param in interface's method

File: src/Application/Interfaces/IRepositories/IGenericRepository.cs (hunk 1 above)
Comment (temshik) [reply src/Application/Interfaces/IRepositories/IGenericRepository.cs:5]: i added GenericRepository to IRepository

Comment (temshik) [reply src/Application/Interfaces/IRepositories/IGenericRepository.cs:5]: We added default values in entity reposisories for tracking params, and in Services we changed GetAll to GetPagedAsync

File: src/Infrastructure/Services/AppUserService.cs (hunk 2 above)
Comment (temshik) [reply src/Infrastructure/Services/AppUserService.cs]: done

File: src/Infrastructure/Services/BookingService.cs (hunk 3 above)
Comment (temshik) [reply src/Infrastructure/Services/BookingService.cs]: created befault values in repositoru methods

**Review by svadrif — APPROVED** [review]

**Pull request merged**